src/com/telesign/util/TeleSignRequest.java

```diff
@@ -497,4 +502,26 @@ private String encode(String data, String key)  throws java.security.SignatureEx
 
 		return result;
 	}
+	
+	/**
+	 * Set the TLS protocol to TLSv1.2
+	 */
+	private void setTLSProtocol() {

```

Comment:
Users who are overriding TLS versions via jvm options will still be able to use TLSv1.1 . As jvm options take precedence over connection specific parameters.

Also please note that any person who has jre7 installed on his system supports both TLSv1.1 & TLSv1.2.

Jre8 uses TLSv1.2 as default.